### ISSUE TITLE
The ethics of Artificial Intelligence

### ISSUE THREAD

**Issue by mia-li-001** [CSCI373group4/Research#9]
[The ethics of Artificial Intelligence | by Justin Lee | The Startup | Medium.pdf](https://github.com/CSCI373group4/Research/files/5168042/The.ethics.of.Artificial.Intelligence.by.Justin.Lee.The.Startup.Medium.pdf)

In this article, the author introduces 4 basic ethic issues: unemployment, inequality, relationship, and singularity. The author also lists some unsolved questions such as "How do we define the humane treatment of AI?" and "How do we eliminate AI bias?"

Unemployment:
A recent Mckinsey report estimates that up to 800 million jobs could be lost worldwide to automation by 2030. If we don't adapt to this, we will ultimately get left behind.

Inequality:
The wealth gap is already widening. Some tech giants create relatively few jobs comparative to their hold on the market. If Silicon Valley can magically provide basic services, from education to health, on its own, why still pay taxes and fund non-existent public services, which are to be provided — on a very different model — by tech companies anyway? 

Relationship:
AI has great potential to transform and support human behavior — as long as it ends up in the right hands. 

Singularity:
Singularity refers to the point at which humans are no longer the most intelligent beings on earth.
Artificial General Intelligence would understand its own design to such an extent that it could redesign itself or create a successor system, which would then redesign itself, and so on, with unknown limits.

How do we define the humane treatment of AI?
If a being has human-like qualities, it earns a moral status. We have an in-built moral obligation to treat it in a certain way, and not in others. Moral status boils down to a two-way criterion: sentience and sapience. Should they be devoid of any moral status, despite their intelligence being higher than that of some humans?

How do we eliminate AI bias?
The capacity of AI processing is far beyond that of humans. But AI can’t be trusted to be fair, or neutral, as it essentially reflects the conscious — or unconscious — biases of its developers (who are usually white males).

In conclusion, the AI is really good at that one thing — but nothing else. The ability to do both from watching and learning is a uniquely human trait. An AGI would be able to learn from watching, and ultimately would be able to accomplish a wide variety of tasks, think and rationalize in the same way a human can — possibly even at a super-human intelligence level.
